libraries/AC_Autorotation/AC_Autorotation.h

```diff
@@ -37,10 +39,44 @@ class AC_Autorotation
     void set_desired_fwd_speed(float speed) { _vel_target = speed; } // Overloaded: Set desired speed to argument value
     int32_t get_pitch(void) const { return _pitch_target; }  // Get pitch target
     float calc_speed_forward(void);  // Calculates the forward speed in the horizontal plane
-    void set_dt(float delta_sec);
+    void set_dt(float delta_sec); // set the loop time
+	void flare_controller();  // Update the flare controller
+	void touchdown_controller();  // Update the touchdown controller
+	void set_ground_distance(float radalt) { _radar_alt = radalt; }
+	void get_entry_speed();
+	float get_ground_distance() const { return _radar_alt; }
+	float get_time_to_ground() const { return _time_to_ground; }
+	void time_to_ground();
+	void set_collective_minimum_drag(float col_mid )  { _col_mid = col_mid; }
+	void set_collective_hover(float col_hover) {_col_hover = col_hover; }
+	void set_collective_max(float col_max) {_col_max = col_max; }
+	void set_collective_min(float col_min) {_col_min = col_min; }
+	void get_gov_rpm(float gov_rpm) {_governed_rpm = gov_rpm; }
```

Comment:
Odd indentation.

Reply:
@IamPete1 I didn't use the style script on the .h files.  I can but I've found when correcting the style in the autotune code that it does some things we don't necessarily like.  I can't remember exactly what but I think it would take any of the "set" methods and use 3 lines to show the code between the braces instead of one line like it typically done.

Reply:
@bnsgeyer I have already fixed up a lot of the style issues in my branch I linked.  I'm not sure if it is easier or not but you could cherry pick the commits across.  Alternatively, I can open a new pr with those changes.

Reply:
thanks @Gone4Dirt ! Picked all your commits and @bnsgeyer has updated the PR